### ISSUE TITLE
Add accessibility information to talks and series

### ISSUE THREAD

**Issue by AndrewVSutherland** [roed314/seminars#343]
We should add a column for accessibility information to both talks and series (talks would inherit the property from their series).  The most common option would be the availability of closed-captioned video recordings posted after the talk, but we should make it clear to organizers that machine generated captions (such as Zoom and YouTube create automatically) do not meet accessibility regulations in most jurisdictions.  Human captioning or editing is required.

Other options might be live-captioning and sign language interpretation (Zoom supports both).  We should then make it possible for users to filter on accessibility options.

**Comment by poonen** [comment]
In addition, we should make sure that our site itself is accessible to the extent possible.

**Comment by AndrewVSutherland** [comment]
@poonen Agreed, but it probably makes sense to create a separate issue for this as the code changes involved (e.g. making the site better suited to screen readers) are really orthogonal to the changes contemplated here.

**Comment by poonen** [comment]
OK, I'll do that.

**Comment by AndrewVSutherland** [comment]
@poonen already done, see #362.
